File: client/Client/Network/Messages/Game/GameInitMessage.cs
using LoadRunnerClient.DTOs;
using System.Collections.Generic;

namespace LoadRunnerClient.Messages
{
    /// Author Florian Ortmann
	public class GameInitMessage
    {
        public const string TYPE = "GameInitMessage";

        private MapDTO _mapDTO;
        private int _minX;
        private int _maxX;
        private List<GameCharacterDTO> _playerDTOs;

        public GameInitMessage()
        {
        }

        public GameInitMessage(MapDTO mapDTO, int minX, int maxX, List<GameCharacterDTO> playerDTOs)
        {
            this._mapDTO = mapDTO;
            this._minX = minX;
            this._maxX = maxX;
            this._playerDTOs = playerDTOs;
        }

        public MapDTO mapDTO
        {
            get => _mapDTO;
            set => _mapDTO = value;
        }

        public int minX
        {
            get => _minX;
            set => _minX = value;
        }

        public int maxX
        {
            get => _maxX;
            set => _maxX = value;
        }

        public List<GameCharacterDTO> playerDTOs

        {
            get => _playerDTOs;
            set => _playerDTOs = value;
        }
    }
}

File: server/src/main/java/de/hsrm/mi/swtpro03/server/network/messages/game/GameInitMessage.java
package de.hsrm.mi.swtpro03.server.network.messages.game;

import de.hsrm.mi.swtpro03.server.game.dtos.GameCharacterDTO;
import de.hsrm.mi.swtpro03.server.game.dtos.MapDTO;

import java.util.List;

public class GameInitMessage {

	public static final String TYPE = "GameInitMessage";

	private MapDTO mapDTO;
	private int minX;
	private int maxX;
	private List<GameCharacterDTO> PlayerDTOs;

	public GameInitMessage() {
	}

	public GameInitMessage(MapDTO mapDTO, int minX, int maxX, List<GameCharacterDTO> gameCharacterDTOS) {
		this.mapDTO = mapDTO;
		this.minX = minX;
		this.maxX = maxX;
		this.PlayerDTOs = gameCharacterDTOS;
	}

	public MapDTO getMapDTO() {
		return mapDTO;
	}

	public void setMapDTO(MapDTO mapDTO) {
		this.mapDTO = mapDTO;
	}

	public int getMinX() {
		return minX;
	}

	public void setMinX(int minX) {
		this.minX = minX;
	}

	public int getMaxX() {
		return maxX;
	}

	public void setMaxX(int maxX) {
		this.maxX = maxX;
	}

	public List<GameCharacterDTO> getPlayerDTOs() {
		return PlayerDTOs;
	}

	public void setPlayerDTOs(List<GameCharacterDTO> gameCharacterDTOList) {
		this.PlayerDTOs = gameCharacterDTOList;
	}
}
